src/components/__tests__/__snapshots__/StateResults.js.snap

```diff
@@ -9,7 +9,7 @@ exports[`renders explanation if no slot is used 1`] = `
   <p>
     Fill in the slot, and get access to the following things on the
     <code>
-      slot-scope
+      v-slot
     </code>
```

Comment:
maybe this should be "Fill in the slot and get access to the following things:"

that way it's not prescribing how to read it, and still straightforward for vue 2 usage

Reply:
I just checked and we do not explicitly say anywhere "requires vue >= 2.6.0", but since we have it in our peerDependencies, can we assume it's being used on vue >= 2.6.0? If that's the case, `v-slot` might be enough.

Reply:
I just don't think "v-slot" or "slot-scope" is the best wording

Reply:
it makes sense https://github.com/algolia/vue-instantsearch/pull/1008/commits/d5d33642fdf1d868925488a21b0a8135cf610499